SourceGitlab/lang/strings_russian.txt

```diff
@@ -0,0 +1,18 @@
+<?php
+
+# Copyright (c) 2014 Johannes Goehr
+# Copyright (c) 2014 Bob Clough
```

Comment:
@okainov you probably want to put your name there, I don't think Johannes or Bob have anything to do with the russian translation ;)

Reply:
Done